src/Streamly/Internal/Data/Parser/ParserK/Types.hs

```diff
@@ -0,0 +1,169 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+#include "inline.hs"
+
+-- |
+-- Module      : Streamly.Internal.Data.Parser.ParserK.Types
+-- Copyright   : (c) 2020 Composewell Technologies
+-- License     : BSD3
```

Comment:
The appropriate SPDX identifier that cabal uses is "`BSD-3-Clause`"

Reply:
Changed. But we need to change all the old files in the source code as well.